### ISSUE TITLE
Fix notSupportedMessage typo, please.

### ISSUE THREAD

**Issue by avreg** [videojs/video.js#3220]
s/video/media/

```
$ diff --git a/src/js/player.js b/src/js/player.js
index 29c421e..61cf6c7 100644
--- a/src/js/player.js
+++ b/src/js/player.js
@@ -2788,7 +2788,7 @@ Player.prototype.options_ = {
   languages: {},
 
   // Default message to show when a video cannot be played.
-  notSupportedMessage: 'No compatible source was found for this video.'
+  notSupportedMessage: 'No compatible source was found for this media.'
 };
```

**Comment by gkatsev** [comment]
Thanks. Can you submit a PR against the stable branch for this? It would make it a lot easier for us to merge it in.

**Comment by avreg** [comment]
Done, #3222.

**Comment by gkatsev** [comment]
Thanks.

**Comment by gkatsev** [comment]
Fixed via #3222.
